### DISCUSSION TITLE
Medals

### DISCUSSION THREAD

**Discussion by Electro-SRS** [ppy/osu#21010] [Q&A] [answered]
It looks like that medals aren't in osu!lazer, is it because it needs to be migrated with stable first or lazer don't have support for medals yet?

**Comment by Doksha** [comment]
lazer is still in development, if medals were enabled people would find ways to cheat the system. There are some pretty recent bugs that would allow cheating medals, so until release I doubt medals or pp will count

**Reply by Electro-SRS** [reply]
Ok

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/18880
